apps/web/src/app/admin-console/organizations/core/services/group/group-api.service.ts

```diff
@@ -46,28 +46,26 @@ export class GroupApiService {
 
     const listResponse = new ListResponse(r, GroupDetailsResponse);
 
-    return (await Promise.all(listResponse.data?.map((gr) => GroupView.fromResponse(gr)))) ?? [];
+    return listResponse.data?.map((gr) => GroupView.fromResponse(gr));
```

Comment:
You still need to remove the optional chaining operator:

```suggestion
    return listResponse.data.map((gr) => GroupView.fromResponse(gr));
```